scripts/deploy/nlp/deploy_triton.py

```diff
@@ -254,13 +263,14 @@ def get_trtllm_deployable(args):
                 nemo_checkpoint_path=args.nemo_checkpoint,
                 model_type=args.model_type,
                 n_gpus=args.num_gpus,
-                tensor_parallel_size=args.num_gpus,
-                pipeline_parallel_size=1,
+                tensor_parallel_size=args.tensor_parallelism_size,
```

Comment:
How about using the same name for this param everywhere: `args.tensor_parallelism_size` -> `args.tensor_parallel_size` (same for `pipeline_parallel_size` below)

Reply:
Good suggestion, thanks. Updated it.